pymeasure/instruments/__init__.py

```diff
@@ -51,4 +51,6 @@ class RangeException(Exception): pass
 from . import srs
 from . import tektronix
 from . import thorlabs
-from . import yokogawa
\ No newline at end of file
+from . import yokogawa
+from resources import list_resources
```

Comment:
The CI is failing because this line needs to a relative import (`from .resources import list_resources`).